### PR TITLE
Add job ad service and message broker

### PR DESCRIPTION
**Pull request by michael-wirth**

### CODE REVIEW

File: group_vars/all [hunk 1]
```diff
@@ -10,24 +10,39 @@ registry:
...
-  maven_repository: https://alvch.jfrog.io/alvch/libs-releases-local
+  maven_repository: https://alvch.jfrog.io/alvch/global
 
   gateway:
     app_name:        gateway
     app_extension:   war
     group_id:        ch.admin.seco.jobroom
     artifact_id:     jobroom
-    startup_timeout: 120
+    startup_timeout: 60
```
Comment (birom) [inline group_vars/all]: On which environment did you test the script?
On staging and prod takes the deployment much more longer as on the test environment. I think for this reason was set the startup_timeout to 120.

Comment (michael-wirth) [reply group_vars/all]: i tested it on test. usually the referenceservice is taking the longest but to be sure lets reset the value

File: group_vars/all [hunk 2]
```diff
@@ -45,7 +60,7 @@ jobroom:
     app_extension:   jar
     group_id:        ch.admin.seco.service.candidate
     artifact_id:     candidateservice
-    startup_timeout: 120
+    startup_timeout: 60
```
Comment (birom) [inline group_vars/all]: Why do we change here the timeout?

File: playbooks/deploy-avam-webservices-untuntu1204.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
```
Comment (birom) [inline playbooks/deploy-avam-webservices-untuntu1204.yml]: There is a typo in the filename.

File: playbooks/deploy-message-broker.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+---
+- name: Install Messagebroker
+  hosts: broker
+  sudo: true
+  environment: "{{ env }}"
+
+  pre_tasks:
+    - name: Install openjdk-8-jdk with default package manager
+      action: "{{backcompat_pkg_mgr}} name=openjdk-8-jdk"
```
Comment (birom) [inline playbooks/deploy-message-broker.yml]: Please add spaces between the variable name and the opening and closing curly braces.

File: group_vars/all (hunk 2 above)
Comment (michael-wirth) [reply group_vars/all]: reverted

File: playbooks/templates/kafka.service.j2 [hunk 5]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=kafka server
+After=network.target
+
+[Service]
+Type=simple
+User={{kafka_user}}
```
[the remark this answers is not in the document]
Comment (michael-wirth) [reply playbooks/templates/kafka.service.j2]: done

File: playbooks/deploy-avam-webservices-untuntu1204.yml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      - "--spring.cloud.dataflow.stream.app.type=sink"
+      - "--spring.cloud.dataflow.stream.app.label=ws-avam"
+      - "--spring.cloud.dataflow.stream.name=ws-avam-export"
+      - "--spring.cloud.stream.bindings.output.destination=jobad.event"
+      - "--spring.cloud.stream.kafka.binder.brokers={{ test.kafka.brokers }}"
+      - "--spring.cloud.stream.kafka.binder.default-broker-port={{ test.kafka.default_broker_port }}"
+      - "--spring.cloud.stream.kafka.binder.zk-nodes={{ test.kafka.zk_nodes }}"
+      - "--spring.cloud.stream.kafka.binder.default-zk-port={{ test.kafka.default_zk_port }}"
+      - "--spring.cloud.application.group=ws-avam-export"
+      - "--jobroom.ws.avam.sink.username=AVAMeGovTechUser"
```
[the remark this answers is not in the document]
Comment (michael-wirth) [reply playbooks/deploy-avam-webservices-untuntu1204.yml]: vault is not working on Ubuntu1204.. any ideas? I remove it and set manually the right password, as we need to start the process manaually. We need to wait for the port oppenings anyway

File: playbooks/deploy-avam-webservices-untuntu1204.yml (hunk 3 above)
Comment (michael-wirth) [reply playbooks/deploy-avam-webservices-untuntu1204.yml]: fixed

**Pull request merged**